src/snmp.c

```diff
@@ -607,6 +655,10 @@ static int csnmp_config_add_host(oconfig_item_t *ci) {
       status = cf_util_get_string(option, &hd->community);
     else if (strcasecmp("Version", option->key) == 0)
       status = csnmp_config_add_host_version(hd, option);
+    else if (strcasecmp("Timeout", option->key) == 0)
```

Comment:
These options need to be added to our documentation before this change can be merged. You need to add it to `src/collectd.conf.pod` and `src/collectd.conf.in`.